Source/WebCore/dom/PointerEvent.cpp

```diff
@@ -36,6 +36,16 @@ namespace WebCore {
 
 WTF_MAKE_ISO_ALLOCATED_IMPL(PointerEvent);
 
+typedef struct PointerEventTilt {
+    long tiltX;
+    long tiltY;
+} PointerEventTilt;
+
+typedef struct PointerEventAngle {
+    double altitudeAngle;
+    double azimuthAngle;
+} PointerEventAngle;
```

Comment:
Why the `typedef struct` construct? This would be equivalent with less typing!

```suggestion
struct PointerEventTilt {
    long tiltX;
    long tiltY;
};

struct PointerEventAngle {
    double altitudeAngle;
    double azimuthAngle;
};
```